### PR TITLE
Feat: create component explore the community

### PR DESCRIPTION
**Pull request by build-ratchanon**
## What does it do?

- create "explore the community" component 

## Screenshot
image

## Related isssue
- #74

### CODE REVIEW

File: yarn.lock
Comment (yoannfleurydev) [inline yarn.lock:1]: Please use https://pnpm.io/

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

File: src/components/explore-the-community.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Button } from "./ui/button";
```
Comment (yoannfleurydev) [inline src/components/explore-the-community.tsx:1]: Please use absolute imports.

```suggestion
import { Button } from "@/components/ui/button";
```

File: src/components/explore-the-community.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <p className="text-xs font-medium">
+        {props.title}
+      </p>
+      <Button size="sm" className="max-h-6 text-xs">
+        Explore the community
+        <ArrowUpRight
+          className="w-3.5 h-3.5 ml-1 inline-block"
+          aria-hidden="true"
+        />
+      </Button>
```
Comment (yoannfleurydev) [inline src/components/explore-the-community.tsx:16]: The button should take optionnal props so we can give it a link (default to `/`)

File: src/components/explore-the-community.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { Button } from "./ui/button";
+import { ArrowUpRight } from "lucide-react";
+
+export const ExploreTheCommunity = (props: Readonly<{ title: string }>) => {
+  return (
+    <div className="flex flex-row justify-between items-center min-h-10 gap-1 p-2 pl-4 bg-[#171717] bg-opacity-40 rounded-md">
+      <p className="text-xs font-medium">
+        {props.title}
+      </p>
```
Comment (yoannfleurydev) [inline src/components/explore-the-community.tsx:9]: I think the `title` props should be optionnal and default to `Fork it! Community`.

File: src/components/explore-the-community.tsx
Comment (yoannfleurydev) [inline src/components/explore-the-community.tsx:1]: Please use Prettier to format the code.

All the config is in the repo, you need to configure your IDE.

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

**Comment by yoannfleurydev** [comment]
Also, is it possible to link the issue to the PR please, so we can keep track of which PR goes with which issue.

Thanks.

File: src/components/explore-the-community.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+export const ExploreTheCommunity = (
+  props: Readonly<{ title?: string; href?: string }>,
+) => {
+  return (
+    <div className="flex min-h-10 flex-row items-center justify-between gap-1 rounded-md bg-[#171717] bg-opacity-40 p-2 pl-4">
+      <p className="text-xs font-medium">
+        {props?.title || "Fork it! Community"}
+      </p>
+      <Button asChild size="sm" className="max-h-6 text-xs">
```
Comment (JeanneGrenet) [inline src/components/explore-the-community.tsx:12]: On the mockups the padding is `8px `but here the padding is `12px`

**Review by JeanneGrenet — CHANGES_REQUESTED** [review]

**Review by JeanneGrenet — APPROVED** [review]

File: src/components/explore-the-community.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import { Button } from "@/components/ui/button";
+import { ArrowUpRight } from "lucide-react";
+
+export const ExploreTheCommunity = (
+  props: Readonly<{ title?: string; href?: string }>,
+) => {
+  return (
+    <div className="flex min-h-10 flex-row items-center justify-between gap-1 rounded-md bg-neutral-900 bg-opacity-40 p-2 pl-4">
```
Comment (houssembaltii) [inline src/components/explore-the-community.tsx:8]:
```suggestion
    <div className="flex min-h-10 flex-row items-center justify-between gap-1 rounded-lg bg-neutral-900 bg-opacity-40 p-2 pl-4">
```

Comment (houssembaltii) [reply src/components/explore-the-community.tsx:8]: according to the [figma design ](https://www.figma.com/design/qt0YXmmXYz0sMx6SYTh84X/branch/4CuDzedIt5n3dLji9FR25d/Digital-Platform?node-id=3462-3410&node-type=frame&m=dev)the border radius is 8 px   .

**Review by yoannfleurydev — APPROVED** [review]

**Pull request merged**